schema/me/update_conversation.js

```diff
@@ -15,20 +16,22 @@ export default mutationWithClientMutationId({
     buyer_outcome: {
       type: new GraphQLNonNull(BuyerOutcomeTypes),
     },
-    id: {
-      type: new GraphQLNonNull(GraphQLString),
+    ids: {
+      type: new GraphQLList(GraphQLString),
     },
   },
-  outputFields: ConversationFields,
-  mutateAndGetPayload: ({ buyer_outcome, id }, request, { rootValue: { accessToken } }) => {
+  outputFields: new GraphQLList(ConversationType),
```

Comment:
This is where we are now returning an array of conversations (from the mutation), except a single one.